homeassistant/components/system_bridge/binary_sensor.py

```diff
@@ -92,6 +91,8 @@ def __init__(
             description.key,
         )
         self.entity_description = description
+        if description.name != UNDEFINED:
```

Comment:
There are no descriptions without names

Reply:
And also, both binary sensors have device classes. I think adding `_attr_has_entity_name = True` to this file would already be sufficient

Reply:
As then the entities will use the device class for naming